normalizer/modules/cowrie_events.py

```diff
@@ -0,0 +1,59 @@
+# Copyright (C) 2012 Johnny Vestergaard <jkv@unixcluster.dk>
```

Comment:
Can you confirm that this is identical to `normalizer/modules/kippo_events.py` with `s/kippo/cowrie/`?  Anything different?